README.md

```diff
@@ -1,57 +1,58 @@
-# A template repo for IIASA Python projects
+# Documentation of the teaching learning material by the Energy, Climate, and Environment Program (ECE)
```

Comment:
Suggest to make a bit shorter, so that it can fit in one line (I hope)
```suggestion
# Overview of the teaching & learning material by the ECE program
```